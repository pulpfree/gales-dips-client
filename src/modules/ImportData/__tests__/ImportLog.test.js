/* eslint no-undef: "off" */

import React from 'react'
import { shallow } from 'enzyme'

import ImportLog from '../ImportLog'

describe('ImportLog', () => {

  let mountedC
  let props
  const reportC = () => {
    if (!mountedC) {
      mountedC = shallow(
        <ImportLog {...props} />
      )
    }
    return mountedC
  }

  beforeEach(() => {
    props = {
      importType: '',
    }
    mountedC = undefined
  })

  it('matches snapshot', () => {
    expect(reportC()).toMatchSnapshot()
  })

})