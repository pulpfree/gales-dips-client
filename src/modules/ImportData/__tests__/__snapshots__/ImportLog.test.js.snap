// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImportLog matches snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <WithStyles(ImportLog)
    importType=""
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "classes": Object {
        "container": "ImportLog-container-1",
        "headerCell": "ImportLog-headerCell-3",
        "headerCellRt": "ImportLog-headerCellRt-4",
        "headerRow": "ImportLog-headerRow-2",
        "reportContainer": "ImportLog-reportContainer-5",
        "reportDataCell": "ImportLog-reportDataCell-7",
        "reportDataRow": "ImportLog-reportDataRow-6",
        "title": "ImportLog-title-8",
      },
      "importType": "",
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "classes": Object {
          "container": "ImportLog-container-1",
          "headerCell": "ImportLog-headerCell-3",
          "headerCellRt": "ImportLog-headerCellRt-4",
          "headerRow": "ImportLog-headerRow-2",
          "reportContainer": "ImportLog-reportContainer-5",
          "reportDataCell": "ImportLog-reportDataCell-7",
          "reportDataRow": "ImportLog-reportDataRow-6",
          "title": "ImportLog-title-8",
        },
        "importType": "",
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
